cpp/include/raft/sparse/solver/detail/lanczos.cuh

```diff
@@ -1815,7 +1815,7 @@ auto lanczos_smallest(
   raft::copy(&res, output.data_handle(), 1, stream);
   resource::sync_stream(handle, stream);
 
-  auto uu  = raft::make_device_matrix<ValueTypeT>(handle, 0, nEigVecs);
+  auto uu  = raft::make_device_matrix<ValueTypeT>(handle, 1, nEigVecs);
```

Comment:
Could you please add a comment here explaining why it should be 0 and not 1 and how setting it to 1 fixes the bug in CUDA 11 without changing the behavior in CUDA 12?

Reply:
> Could you please add a comment here explaining why it should be 0 and not 1 and how setting it to 1 fixes the bug in CUDA 11 without changing the behavior in CUDA 12?

Basically, it should be `1` according to the cuBLAS documentation, which doesn’t mention `y` being `nullptr`. So, a separate comment explaining potential misusage isn’t necessary, right? :grimacing:

Reply:
Just for hindsight, a comment with a link to the docs is appreciated :)